### PR TITLE
nc-news ticket #3

### PR DESCRIPTION
**Pull request by Clare30**
test suite set up along with test for ticket #3 
controllers and models folder set up with an index.js file and the first controller & model function for ticket #3

tests passing for get all topics request and error handling

### CODE REVIEW

File: controllers/topics.controller.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const { selectTopics } = require("../models/topics.model.js");
+
+exports.getTopics = (req, res, next) => {
+  selectTopics()
+    .then((topics) => {
+      res.status(200).send(topics);
+    })
```
Comment (daviidbartlett) [inline controllers/topics.controller.js:7]: you'll likely want to make your response to be an object. eg `{topics: [...] }`

**Review by daviidbartlett — COMMENTED** [review]
just one change 😄

File: app.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+const app = express();
+
+module.exports = app;
+
+app.get("/api/topics", controllers.topics.getTopics);
+
+app.all("/*", (req, res) => {
+  res.status(404).send({ msg: "path does not exist" });
+});
```
Comment (Angaca) [inline app.js:12]: You may want to move error handling in a separate file as well

**Review by Angaca — APPROVED** [review]
Good to go!

**Pull request merged**